CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 # Changelog
 All notable changes to this project will be documented in this file. 
 
+
+## [2.0.x] - 2020-12-02
+
+### Changed
+- BFV : plaintext must not be created using NewPlaintextZQ, NewPlaintextZT or NewPlaintextMul. See bfv/plaintext.go for the full details.
+- BFV : QiMul is now automatically generated by the evaluator
```

Comment:
The important thing is not where it is generated , but that it is controlled by the user in a different way.

I would simply add that it is computed from X and Y after the corresponding entry in the "Removed" section below.
Also please  specify use the proper type names to improved readability. E.g., removed field `QiMul` from `bfv.Parameters`, it is now computed directly from X and Y